src/ftsh_functions.c

```diff
@@ -1,130 +1,176 @@
 #include <stdlib.h>
+#include <time.h>
 #include "ftsh.h"
 #include "ftsh_functions.h"
 #include "search.h"
 #include "match.h"
+#include "trie.h"
 #include "../src/simclist.h"
 
 
-/* 
+/*
     Defined variables for greater readability
-*/
+ */
 #define SHOULD_CONTINUE 1
 #define SHOULD_EXIT 0
 
 /*
     List of builtin commands, followed by their corresponding functions.
  */
 char *builtin_str[] = {
-    "help",
-    "exit",
-    "find"
+	"help",
```

Comment:
This file appears to use tabs for indentation. Please replace every tab character with 4 spaces.